proto/api/v1/notification_service.proto

```diff
@@ -0,0 +1,117 @@
+syntax = "proto3";
+
+option go_package = "v1";
+
+option java_package = "io.stackrox.proto.api.v1";
+
+import weak "google/api/annotations.proto";
+import "api/v1/common.proto";
+import "api/v1/pagination.proto";
+import "google/protobuf/timestamp.proto";
+
+package v1;
+
+message NotificationsResponse {
+    repeated Notification notifications = 1;
+    Pagination              pagination  = 2;
```

Comment:
strange formatting, I would either do no whitespace after `Pagination` or align it with `notification`.